src/pages/login/CertificateAdd.tsx

```diff
@@ -1,13 +1,15 @@
 import React, { FC } from "react";
-import { Col, Row } from "@canonical/react-components";
+import { Col, Notification, Row } from "@canonical/react-components";
 import { Navigate } from "react-router-dom";
 import { useAuth } from "context/auth";
 import Loader from "components/Loader";
-import CertificateAddForm from "pages/certificates/CertificateAddForm";
+import CertificateAddForm from "pages/login/CertificateAddForm";
 import NotificationRow from "components/NotificationRow";
+import { useNotify } from "context/notify";
 
 const CertificateAdd: FC = () => {
   const { isAuthenticated, isAuthLoading } = useAuth();
+  const notify = useNotify();
```

Comment:
What is this for? It seems unused to me.

Reply:
We use it to either show the notification, or the warning, that a cert needs to be imported for this to work.

Reply:
It's not needed for the warning - I've just removed the `notify` and it still displays, since it's independent - and I don't see which other notification we could show in this context.

Reply:
When either not having a cert in the browser (or select) or using an invalid token, there will be an error message.

Reply:
Right, I see the use case now. Thanks.